Jenkinsfile

```diff
@@ -76,6 +76,8 @@ pipeline {
               "^tests/packaging.*",
               "^vendor/github.com/elastic/beats.*"
             ]
+            // It requires to use /dev/null with tail to avoid the mage output installation
+            setEnvVar('APM_SERVER_VERSION', sh(label: 'Get beat version', script: 'make get-version 2>/dev/null | tail -n 1', returnStdout: true)?.trim())
```

Comment:
**NOTE** BEAT_VERSION is already defined somewhere therefore we cannot set it as we have done for the beats pipeline

Reply:
and this approach won't work


```
## To be consumed by the beats-tester pipeline
COMMIT=e10a74195c4ff4310fd150090a06d359cb75bfbb
APM_URL_BASE=https://storage.googleapis.com/apm-ci-artifacts/jobs/commits/e10a74195c4ff4310fd150090a06d359cb75bfbb
VERSION=Makefile:242: recipe for target 'build//bin/mage' failed-SNAPSHOT
```